src/snowflake/snowpark/_internal/telemetry.py

```diff
@@ -168,6 +168,10 @@ def wrap(*args, **kwargs):
         ]._session.sql_simplifier_enabled
```

Comment:
seems we are also missing the paramter value for cte enabled and large query enabled

Reply:
they are added when their values are changed and in compiler statistics